content/vendor/guides/production-ready.md

```diff
@@ -12,7 +12,7 @@ It is split into three parts:
 
 - [Implementation Steps](#implementation-steps)
 - [Further Reading](#further-reading)
-- [Appendix A: A Questionnaire for your End Users](#appendix-a-a-questionnaire-for-your-end-users)
+- [Appendix: A Questionnaire for your End Users](#appendix-a-a-questionnaire-for-your-end-users)
```

Comment:
I think you need to change the link from: 
`#appendix-a-a-questionnaire-for-your-end-users`
to 
`#appendix-a-questionnaire-for-your-end-users`

Reply:
d'oh -- thanks. Thanks HTML linter!